config/flipt.schema.cue

```diff
@@ -115,7 +115,15 @@ import "strings"
 	}
 
 	#authorization: {
-		required?: bool | *false
+		#authorizationSource: {
+			backend: "filesystem"
```

Comment:
What about 'local' to match storage for flag state?

Reply:
We can do that, do we want to change the recent addition to the `storage.git.backend` to also be `local`?
I was making this consistent with that.